vizro-core/docs/pages/examples/examples.md

```diff
@@ -13,14 +13,14 @@ The dashboard launches with a home page that offers four other pages:
 * Continent summary: Summarizes the main findings for each continent.
 
 
-You can find the code for each of the charts, for each page of the dashboard, in the `examples` folder of the `vizro-core` package, within [Vizro's GitHub repository](https://github.com/mckinsey/vizro). The code is available as a [`.py` file](https://github.com/mckinsey/vizro/blob/main/vizro-core/examples/demo/app.py) or as a [Jupyter Notebook](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples/demo/jupyter_version).
+You can find the code for each of the charts, for each page of the dashboard on our [huggingface gapminder demo space](https://huggingface.co/spaces/vizro/Main_Demo). The code is available as a [`.py` file](https://huggingface.co/spaces/vizro/Main_Demo/blob/main/app.py).
```

Comment:
@maxschulz-COL - could you let me know when you have renamed the demo to gapminder? As discussed in our thread :) I'll update the name here then 👍

Reply:
Never mind, I've just done it 😄

Reply:
I tried out a few things, and now they are called slightly differently to be all in line. Will make a suggestion for easy update